zipkin-server/pom.xml

```diff
@@ -447,13 +447,31 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>${git-commit-id.version}</version>
+        <executions>
+          <execution>
+            <id>extract-git-info</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
```

Comment:
ah I see. it I suppose wasn't a runtime dep responsible for the verbosity, but rather spring boot detecting this file. As format of things like this have drifted on us, probably another good reason to opt out. I do think this is the right plugin, though.